packages/mdc-menu/simple/adapter.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Provides type information for the adapter to the Closure compiler.
```

Comment:
This documentation should say more about what SimpleMenu is, and what it's adapter has to handle.